LauraBeatris/use-selected-items-hook
# Tests and minor improvements 🎉

Laura, I took the liberty to start testing the library, if there is no problem with it, follow the change bellow :)

#### Changes proposed in this Pull Request:

- Creation of tests (About test reducer https://testing-library.com/docs/example-react-hooks-usereducer/)
- Minor change on `initialItems` type
- Turn off eslint error for dev dependencies

#### Testing instructions:

As it is a test construction you can test it running yarn test.
You can also run `yarn test --coverage` to check it coverage

#### Proposed changelog entry for your changes:

- Use Selected Items Hook: set initialItems as optional since it has a default value `[]`
- Tests: add test to library
